### DISCUSSION TITLE
ODBC Source

### DISCUSSION THREAD

**Discussion by axelbehappy** [airbytehq/airbyte#40066] [Connector Questions]
Hi dear Community,

I am new to airbyte. Due to a DWH-project we'll have to connect an old ERP system via ODBC. I didn't find a connector for Airbyte. What's the way to connect airbyte to an ODBC source?

Thanks a lot in advance!

All the best
Axel

**Comment by nathanfreystaetter** [comment]
We have the same need. Did you find an answer to this?

**Comment by obdi70** [comment]
Hello all, 
I´m also new in this topic here, but with same need to connect to on-premise systems. In my case to databases of SAP systems.
 
Can someone please explain why there is not a generic ODBC (or JDBC) source (template) available? 

In fact I would assume that e.g. MS SQL or Oracle can be used as a blue print for a connector because the ODBC and JDBC drivers are available here https://tools.hana.ondemand.com/#hanatools. Or is this just simple minded?

Br.,

[a comment by an automated account is not shown]
